lib/Frontend/ModuleInterfaceSupport.cpp

```diff
@@ -131,8 +131,12 @@ static void printImports(raw_ostream &out,
     if (Opts.PrintSPIs) {
       llvm::SmallSetVector<Identifier, 4> spis;
       M->lookupImportedSPIGroups(importedModule, spis);
-      for (auto spiName : spis)
-        out << "@_spi(" << spiName << ") ";
+      for (Identifier spiName : spis) {
```

Comment:
Does this print a separate `@_spi` attribute for each name? I thought we accepted `@_spi(Foo, Bar)`—am I wrong about that?

Reply:
It does print a separate attribute for each name. The first design did support many names per attribute when it was in the form of `public(Foo, Bar)`, however it's not currently supported with `@_spi(Foo)`. I preferred to keep extra arguments for possible options, like adding an `open` or availability. The backing implementation is still present in the compiler so it can be brought back if there's a demand for it.